include/AlgTraits.h

```diff
@@ -120,10 +120,9 @@ struct AlgTraitsHexGL {
 struct AlgTraitsEdge_3D
 {
   static constexpr int nDim_ = 3;
-  static constexpr int nodesPerElement_ = 3;
-  static constexpr int numScsIp_ = 2;
-  static constexpr int numScvIp_ = 0;
-  static constexpr int numGp_ = 2; // for FEM
+  static constexpr int nodesPerElement_ = 2;
+  static constexpr int numScsIp_ = 1;
+  static constexpr int numScvIp_ = 2;
```

Comment:
I do not think that numScvIp_ is required, however, you need not remove this prior to push.